index.bs

```diff
@@ -3474,32 +3474,32 @@ determine what Web IDL language feature to use:
 
     Another consideration is that the prose for [=overloaded=] operations can be specified in
     separate blocks, which can aid in both reading and writing specifications. This is not the case
-    for [=optional arguments=]. This means that in the first case the specification author may write
+    for [=optional arguments=]. This means that in the first case the specification author can write
     the prose definition of the operations as:
 
     <div algorithm="execute supports() with overloads">
 
         The <code class="idl">supports(<var ignore>property</var>, <var ignore>value</var>)</code>
-        method, when called, must run these steps:
+        method, when called, takes the following steps:
 
         1.  …
 
         ----
 
         The <code class="idl">supports(<var ignore>conditionText</var>)</code> method, when called,
-        must run these steps:
+        takes the following steps:
 
         1.  …
 
     </div>
 
-    Yet using <var ignore>value</var> as an [=optional argument=], the specification author must use
-    more boilerplate-style text to effectively replicate the [=overload resolution algorithm=].
+    Yet using <var ignore>value</var> as an [=optional argument=], the specification author has to
+    use more boilerplate-style text to effectively replicate the [=overload resolution algorithm=].
 
     <div algorithm="execute supports() with optional argument">
 
         The <code class="idl">supports(|propertyOrConditionText|, |value|)</code> method, when
-        called, must run these steps:
+        called, takes the following steps:
```

Comment:
I think in this case it would actually be acceptable to use must, since it's an example.